lib/helpers/multiSigInterfaceHelper.ts

```diff
@@ -0,0 +1,151 @@
+}
+
+/// Confirm proposal
+export async function confirmProposal(
+  multiSig: MultiSig,
+  proposalId: number,
+  signer: SignerWithAddress
+): Promise<ContractReceipt> {
+  const tx = await multiSig.connect(signer).confirmProposal(proposalId);
+  return await tx.wait();
+}
+
+/// Revoke confirmation
+export async function revokeConfirmation(
+  multiSig: MultiSig,
+  proposalId: number,
+  signer: SignerWithAddress
+) {
+  const tx = await multiSig.connect(signer).revokeConfirmation(proposalId);
+  return await tx.wait();
+}
+
+/// Schedule proposal
+export async function scheduleProposal(
+  multiSig: MultiSig,
+  proposalId: number,
+  signer: SignerWithAddress
+) {
+  const tx = await multiSig.connect(signer).scheduleProposal(proposalId);
+  return await tx.wait();
+}
+
+/// Execute proposal
+export async function executeProposal(
+  multiSig: MultiSig,
+  proposalId: number,
+  signer: SignerWithAddress
+) {
+  const tx = await multiSig.connect(signer).executeProposal(proposalId);
+  return await tx.wait();
+}
+
+/// Get proposal
+export async function getProposal(
+  multiSig: MultiSig,
+  proposalId: number
+): Promise<
+  [string[], BigNumber[], string[]] & {
+    destinations: string[];
+    values: BigNumber[];
+    payloads: string[];
+  }
+> {
+  return await multiSig.getProposal(proposalId);
+}
+
+/// Get confirmations
+export async function getConfirmations(multiSig: MultiSig, proposalId: number): Promise<string[]> {
+  return await multiSig.getConfirmations(proposalId);
+}
+
+/// Is proposal fully confirmed?
+export async function isFullyConfirmed(multiSig: MultiSig, proposalId: number): Promise<boolean> {
+  return await multiSig.isFullyConfirmed(proposalId);
+}
+
+/// Is proposal scheduled?
+export async function isScheduled(multiSig: MultiSig, proposalId: number): Promise<boolean> {
+  return await multiSig.isScheduled(proposalId);
+}
+
+/// Get timestamp
+export async function getTimestamp(multiSig: MultiSig, proposalId: number): Promise<BigNumber> {
+  return await multiSig.getTimestamp(proposalId);
+}
+
+/// Is proposal time-lock reached?
+export async function isProposalTimelockReached(
+  multiSig: MultiSig,
+  proposalId: number
+): Promise<boolean> {
+  return await multiSig.isProposalTimelockReached(proposalId);
+}
+
+/// Is given address an owner of multisig contract?
+export async function isOwner(multiSig: MultiSig, address: string): Promise<boolean> {
+  return await multiSig.isOwner(address);
+}
+
+/// Is proposal confirmed by address?
+export async function isConfirmedBy(
+  multiSig: MultiSig,
+  proposalId: number,
+  address: string
+): Promise<boolean> {
+  return await multiSig.isConfirmedBy(proposalId, address);
+}
+
+/// Parse events emitted by contract functions
+export function parseEvents(receipt: ContractReceipt, eventName: string) {
+  const event = receipt.events?.find((event) => event.event === eventName);
+  console.log("new event emitted:", event?.event, `(${event?.args})`);
```

Comment:
do we really need the console log?

Reply:
This console log is to provide more information to the user about the transaction they executed. Showing the events that are emmitted after the transaction is a good way of verifying that the transaction succeeded. Do you have a prefered way of providing feedback to the user after they submit a TX? I could use `chalk` to make it colorfull